main.go

```diff
@@ -13,8 +14,9 @@ import (
 )
 
 var (
-	app     *cli.App
-	signals chan os.Signal
+	app         *cli.App
+	signals     chan os.Signal
+	sqsDefaults sqsSettings
```

Comment:
Since the SQS client is really owned by [Queue](https://github.com/enova/scout/blob/main/queue.go#L38), I think it'd be cleaner to add these properties onto the [Config](https://github.com/enova/scout/blob/main/config.go) object and let Queue handle passing them down to SqsClient at the time it [initializes](https://github.com/enova/scout/blob/main/queue.go#L38) it. Passing them under-the-table with this global variable is breaking the encapsulation.

I don't have a strong opinion on if these _need_ to come from the config file as opposed to coming in from ENV vars, but if you add these `sqsSettings` into the `config` object (e.g. `config.SQS`), then I think we'll get the ability to set them from the config file for free which will fit better into the way that Enova uses Scout.

Reply:
Let me know if the updated commit does what you're thinking. Ideally I want to make the config file entirely optional and allow everything to be set via environment